src/renderer/components/+workloads-replicasets/replicasets.tsx

```diff
@@ -8,16 +8,13 @@ import "./replicasets.scss";
 import React from "react";
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency